python/lsst/faro/measurement/TractMeasurementTasks.py

```diff
@@ -51,15 +51,14 @@ def run(
         self, metricName, catalogs, photoCalibs=None, astromCalibs=None, dataIds=None
```

Comment:
Why aren't these parameters documented?

Reply:
We need to add documentation in several places. I think documenting the parameters in the measurement task run methods is a high priority. Created separate ticket here:

https://jira.lsstcorp.org/browse/DM-31473

Reply:
Okay. It's hard for me to understand what the `catalogs` parameter is going to be in order to work out why `"u" in catalogs` is going to work.